project/MinnieKenny.scala

```diff
@@ -1,7 +1,7 @@
 import sbt.Keys._
 import sbt._
 import complete.DefaultParsers._
-import sbt.ProcessLogger
+import scala.sys.process._
```

Comment:
Any reason this isn't `import scala.sys.process.Logger`?

Reply:
That does not compile and it did not seem worthwhile figuring out why